src/trace_processor/perfetto_sql/stdlib/android/surfaceflinger.sql

```diff
@@ -0,0 +1,86 @@
+--
+-- Copyright 2025 The Android Open Source Project
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     https://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+CREATE PERFETTO TABLE _android_sf_process AS
+SELECT
+  *
+FROM process
+WHERE
+  process.name = '/system/bin/surfaceflinger'
+LIMIT 1;
+
+CREATE PERFETTO TABLE _android_sf_main_thread AS
+SELECT
+  upid,
+  utid,
+  thread.name,
+  thread_track.id AS track_id
+FROM thread
+JOIN _android_sf_process AS sf_process
+  USING (upid)
+JOIN thread_track
+  USING (utid)
+WHERE
+  thread.is_main_thread;
+
+-- Extract thread and track information for a given thread in the surfaceflinger process.
+CREATE PERFETTO FUNCTION android_sf_thread(
+    -- thread_name to fetch information for.
+    thread_name STRING
+)
+RETURNS TABLE (
+  -- upid of the process.
+  upid JOINID(process.id),
+  -- utid of the process.
+  utid JOINID(thread.id),
+  -- name of the thread.
+  name STRING,
+  -- track_id for the thread.
+  track_id JOINID(track.id)
+) AS
+SELECT
+  upid,
+  utid,
+  thread.name,
+  thread_track.id AS track_id
+FROM thread
+JOIN _android_sf_process AS sf_process
+  USING (upid)
+JOIN thread_track
+  USING (utid)
+WHERE
+  thread.name = $thread_name;
+
+-- Match the frame timeline on the app side with the frame timeline on the SF side.
+-- In cases where there are multiple layers drawn, there would be separate frame timeline
+-- slice for each of the layers. GROUP BY is used to deduplicate these rows.
+CREATE PERFETTO TABLE android_app_to_sf_vsync_match AS
```

Comment:
Please improve the table name here. It's very cryptic what's going on, I would not understand that this was about frames from looking at the name at a glance.